bundles/core/src/main/java/com/adobe/cq/commerce/core/components/models/product/VariantValue.java

```diff
@@ -25,9 +27,17 @@
 public interface VariantValue {
     String getLabel();
 
+    default String getDefaultLabel() {
```

Comment:
Please use a more descriptive name that matches the purpose of the value: `getCssClassName()`

Reply:
In this case we use it for class modifiers. but that is not the purpose of this field. we might end-up using this for something different in the future. I would keep it as is

Reply:
If we would use it as is (without any logic applied) I agree. In this particular case however we change the default label value in order to make it usable as BEM modifier in a css class name. If we leave it with `getDefaultLabel()` and we would want to expose the real default label in the future, how would you name the method? 

Let's find a better naming that reflects what the values is meant for.